python/lsst/summit/utils/quickLook.py

```diff
@@ -224,7 +234,17 @@ def run(self, ccdExposure, *,
             isrConfig.doCrosstalk = True
             self.log.info("Running with crosstalk correction")
 
-        if bfKernel:
+        if newBFKernel is not None:
+            try:
+                bfGains = newBFKernel.gain
+                isrConfig.doBrighterFatter = True
+                self.log.info("Running with new brighter-fatter correction")
+            except AttributeError:
+                bfGains = None
```

Comment:
If a `newBFKernel` is supplied but we fail to pull the gains out then I'd think we'd probably want to at least log a warning here. What's your thinking on why this shouldn't be a `raise` though?

Reply:
Maybe I took the statement "Should never fail" in the docstring of this task too literally! I'm happy to either log a warning or let it raise, your choice.

Reply:
Ah, yeah, I see, I did wonder if it was something like that. I mean, it should never fail based if valid things are being passed in, but if, say, a calib was the wrong size or something truly awful like that then I think in _those_ cases it should still raise tbh, otherwise the behaviour is sort of ill-defined. Everything that's passed in should work, and if it doesn't then I think that's raise-time. All the other raises like that are implicit and come from isrTask raising though, so I think raising here is fine.